install.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Use environment variables VERSION and BRANCH, with defaults if not set
+version=${VERSION:-$(curl -s https://api.github.com/repos/twentyhq/twenty/releases/latest | grep '"tag_name":' | cut -d '"' -f 4)}
+branch=${BRANCH:-main}
+
+echo "🚀 Using version $version and branch $branch"
+
+# Create a directory named Twenty
+echo "📁 Creating directory 'Twenty'"
+mkdir -p Twenty && cd Twenty
```

Comment:
Feels a little unusual to use capitalized letter. "twenty" instead of "Twenty"?

Reply:
Also if folder already exists (for example in my case because I had git cloned the repo), maybe we want to generate a different folder name or ask for a folder name?

E.g.
```
dir_base_name="twenty"
dir_name="$dir_base_name"
counter=0

# Check if the directory already exists and find a unique name
while [ -d "$dir_name" ]; do
  let counter+=1
  dir_name="${dir_base_name}_${counter}"
done

# Create and enter the new, unique directory
mkdir -p "$dir_name" && cd "$dir_name" || { echo "Failed to create/access directory '$dir_name'"; exit 1; }
```

Or even better:
```

#!/bin/bash
dir_name="twenty"

while [ -d "$dir_name" ]; do
  read -p "Directory '$dir_name' already exists. Please enter a new directory name: " user_input
  dir_name=$user_input  
done

mkdir -p "$dir_name" && cd "$dir_name" || { echo "Failed to create/access directory '$dir_name'"; exit 1; }
```